extensions/stackdriver/edges/TODO

```diff
@@ -0,0 +1,7 @@
+(P0) Figure out mesh_uid data source
```

Comment:
nit: open an issue/epic instead

Reply:
Updated code to grab this from nodeInfo.